addons/l10n_zm/data/template/account.account-zm.csv

```diff
@@ -0,0 +1,87 @@
+"zm_account_4560000","License costs","4560000","expense","False"
+"zm_account_4600000","Telephone & Fax","4600000","expense","False"
+"zm_account_4650000","Travel & Accommodation","4650000","expense","False"
+"zm_account_4700000","Loss on Sale of Non Current Assets","4700000","expense","False"
+"zm_account_5200000","Retained Income / (Accumulated Loss)","5200000","liability_current","False"
+"zm_account_5500000","Long Term Liabilities","5500000","liability_non_current","False"
+"zm_account_5600000","Instalment Sale Creditors Other","5600000","liability_non_current","False"
+"zm_account_6100000","Land & Buildings - Net Value","6100000","asset_fixed","False"
+"zm_account_6100010","Land & Buildings - At Cost","6100010","asset_fixed","False"
+"zm_account_6100020","Land & Buildings - Accumulated Depreciation","6100020","asset_fixed","False"
+"zm_account_6200000","Motor Vehicles - Net Value","6200000","asset_fixed","False"
+"zm_account_6200010","Motor Vehicles - At Cost","6200010","asset_fixed","False"
+"zm_account_6200020","Motor Vehicles - Accumulated Depreciation","6200020","asset_fixed","False"
+"zm_account_6250000","Computer Equipment - Net Value","6250000","asset_fixed","False"
+"zm_account_6250010","Computer Equipment - At Cost","6250010","asset_fixed","False"
+"zm_account_6250020","Computer Equipment - Accumulated Depreciation","6250020","asset_fixed","False"
+"zm_account_6300000","Office Equipment - Net Value","6300000","asset_fixed","False"
+"zm_account_6300010","Office Equipment - At Cost","6300010","asset_fixed","False"
+"zm_account_6300020","Office Equipment - Accumulated Depreciation","6300020","asset_fixed","False"
+"zm_account_6350000","Furniture & Fittings - Net value","6350000","asset_fixed","False"
+"zm_account_6350010","Furniture & Fittings - At Cost","6350010","asset_fixed","False"
+"zm_account_6350020","Furniture & Fittings - Accumulated Depreciation","6350020","asset_fixed","False"
+"zm_account_6600000","Other Fixed Assets - Net Value","6600000","asset_fixed","False"
+"zm_account_6600010","Other Fixed Assets - At Cost","6600010","asset_fixed","False"
+"zm_account_6600020","Other Fixed Assets - Accumulated Depreciation","6600020","asset_fixed","False"
+"zm_account_7000000","Goodwill / Other Intangible Assets","7000000","asset_fixed","False"
+"zm_account_7100000","Investments","7100000","asset_current","False"
+"zm_account_7700000","Inventory Control Account","7700000","liability_current","False"
+"zm_account_8000000","Customers","8000000","asset_receivable","True"
+"zm_account_8100000","Customes (POS)","8100000","asset_receivable","True"
+"zm_account_8200000","Sundry Customers","8200000","asset_current","False"
+"zm_account_8300000","VAT Purchase Tax Control Account","8300000","asset_current","False"
+"zm_account_8900000","Deferred Expenses","8900000","asset_current","True"
+"zm_account_9000000","Suppliers","9000000","liability_payable","True"
+"zm_account_9100000","GRN Accrual Account","9100000","liability_payable","True"
```

Comment:
this is payable?

Reply:
Yes, Goods Received Note are goods not bought but not invoiced yet in my understanding. Looks like a payable to me. Might be wrong tho.